enarx-keep/shims/shim-sev/src/hostlib.rs

```diff
@@ -43,6 +43,12 @@ use nbytes::bytes;
 #[allow(clippy::integer_arithmetic)]
 const ALIGN_SECTION: usize = bytes!(1; MiB);
 
+/// Enarx syscall extension: get `MemInfo` from the host
+pub const SYS_ENARX_MEM_INFO: i64 = 33333;
```

Comment:
I would also want this to be in an explicit sub-module like

```rust
pub mod syscall_ext {
   /// Enarx syscall extension: get `MemInfo` from the host
   pub const SYS_ENARX_MEM_INFO: i64 = 33333;
}
```

though that may not be useful.

Reply:
Will do, if it grows too much